blocks/post/edit.tsx

```diff
@@ -92,6 +95,47 @@ export default function Edit({
     });
   };
 
+  const isShown = (item: string) => {
+    switch (item) {
+      case 'image':
+        return showImage;
+      case 'title':
+        return true;
+      case 'byline':
+        return showByline;
+      case 'excerpt':
+        return showExcerpt;
+      case 'content':
+        return showContent;
+      case 'cta':
+        return showCta;
+      default:
+        return false;
+    }
+  };
+
+  const toggleShown = (item: string) => {
+    switch (item) {
+      case 'image':
+        setAttributes({ showImage: !showImage });
+        break;
+      case 'byline':
+        setAttributes({ showByline: !showByline });
+        break;
+      case 'excerpt':
+        setAttributes({ showExcerpt: !showExcerpt });
+        break;
+      case 'content':
+        setAttributes({ showContent: !showContent });
+        break;
+      case 'cta':
+        setAttributes({ showCta: !showCta });
+        break;
+      default:
+        break;
+    }
+  };
```

Comment:
I went back and forth on this but something like this may be cleaner, but really not sure:

```
const displayPreferences = {
     'image': showImage,
     'byline': showByline,
     'excerpt': showExcerpt,
     'content': showContent,
     'cta': showCta,
     'title': true
  };

  const isShown = (item: string) => displayPreferences[item] ?? false;
  
  const toggleShown = (item: string) => {
    setAttributes((prevAttributes: EditProps["attributes"]) => ({
      ...prevAttributes,
      [`show${item.charAt(0).toUpperCase() + item.slice(1)}`]: !prevAttributes[`show${item.charAt(0).toUpperCase() + item.slice(1)}`]
    }));
  };

```

Reply:
I like this suggestion, but this work should be temporary (see https://github.com/alleyinteractive/wp-newsletter-builder/issues/48) so I'm going to leave it as is for now.